images/mariadb/Dockerfile

```diff
@@ -27,6 +27,7 @@ do mkdir -p $i; chown mysql $i;  /bin/fix-permissions $i; done
 
 COPY docker-entrypoint.sh /lagoon/entrypoints/90-mariadb-entrypoint
 COPY mysql-backup.sh /lagoon/
+RUN chmod 755 /lagoon/mysql-backup.sh
```

Comment:
I have found that just setting executeable and committing a script is enough to make it work in the container, eliminating another step from the Dockerfile.